types.ts

```diff
@@ -788,6 +802,26 @@ export interface AWSAssumeRoleAuthentication extends BaseAuthentication {
   service: string;
 }
 
+/**
+ * Only for use by Coda-authored packs.
+ *
+ * @hidden
+ */
+export interface GoogleDomainWideDelegationAuthentication extends BaseAuthentication {
```

Comment:
Do we want scopes for both of these? At least for DWD the admin needs to assign the scopes the pack needs to the service account?

Reply:
Sure, added.